File: src/main/java/com/mrpowergamerbr/loritta/userdata/JoinLeaveConfig.java
package com.mrpowergamerbr.loritta.userdata;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class JoinLeaveConfig {
	public boolean isEnabled = false;
	public boolean tellOnJoin = true;
	public boolean tellOnLeave = true;
	public String joinMessage = "👉 {@user} entrou no servidor!";
	public String leaveMessage = "👈 {nickname} saiu do servidor!";
	public String canalJoinId = null;
	public String canalLeaveId = null;
}

File: src/main/java/com/mrpowergamerbr/loritta/userdata/JoinLeaveConfig.kt
package com.mrpowergamerbr.loritta.userdata

class JoinLeaveConfig {
	@AllowReflection
	var isEnabled: Boolean = false
	@AllowReflection
	var tellOnJoin: Boolean = true
	@AllowReflection
	var tellOnLeave: Boolean = true
	@AllowReflection
	var joinMessage: String = "\uD83D\uDC49 {@user} entrou no servidor!"
	@AllowReflection
	var leaveMessage: String = "\uD83D\uDC48 {nickname} saiu do servidor!"
	@AllowReflection
	var canalJoinId: String? = null
	@AllowReflection
	var canalLeaveId: String? = null
	@AllowReflection
	var tellOnPrivate: Boolean = false
	@AllowReflection
	var joinPrivateMessage: String = "Obrigado por entrar na {guild} {@user}! Espero que você curta o nosso servidor!"
	@AllowReflection
	var tellOnBan: Boolean = false
	@AllowReflection
	var banMessage: String = ""
	@AllowReflection
	var tellOnKick: Boolean = false
	@AllowReflection
	var kickMessage: String = ""
}